src/utils/conversion.ts

```diff
@@ -1437,7 +1437,7 @@ export function authorizationDataToAuthorization(
 /**
  * @hidden
  */
-export function stringToMemo(value: string, context: Context): Memo {
+export function stringToMemo(value: string, context: Context): PolymeshCommonUtilitiesBalancesMemo {
```

Comment:
This isn't strictly needed for the fix, but is one less schema type to worry about. I tested `transferPolyx` method with this as well.